scripts/jenkins-yoctobuild-build.sh

```diff
@@ -394,21 +395,22 @@ export GOPATH="$WORKSPACE/go"
 )
 
 if grep mender_servers <<<"$JOB_BASE_NAME"; then
-    # Use release tool to query for available repositories, and fall back to
-    # flat list for branches where we don't have that option.
-    for build in $($WORKSPACE/integration/extra/release_tool.py --list -a 2>/dev/null \
-                          || echo "deployments deviceadm deviceauth gui inventory mender-api-gateway-docker useradm" ); do (
-
-        case "$build" in
-            deployments*|deviceadm|deviceauth|inventory|tenantadm|useradm*)
-                cd go/src/github.com/mendersoftware/$build
+    # Use release tool to query for available docker names.
+    for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker -a ); do (
+
+        git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
+        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+
+        case "$docker" in
+            deployments*|deviceadm*|deviceauth*|inventory*|tenantadm*|useradm*)
```

Comment:
I was thinking of it as future-proofing: When we make new enterprise services they will "automagically" work. Not sure if it's smart or naive... I can remove it if you think it should be more explicit, otherwise I will leave it as is.

Reply:
I see your point and is a valid one. At least we should remove it for `tenantadm`, making this way the criteria like "all OS components might have a enterprise version". I'll be happy with that.

I don't have a very good reason for it to be explicit, I guess is just personal taste: if I create a new service and it works out of the box I personally will be very skeptical and think that I did something wrong... hehe.

Reply:
Ok, good enough reason for me. I will change it. :-)

Reply:
All service matching is now explicit, no wildcards (except for `mender-client-qemu*`)